### PR TITLE
9599 change registering party

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#9599

*Description of changes:*
- rework secured party component to allow change of registering party
- tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/parties/RegisteringParty.vue [hunk 1]
```diff
@@ -124,13 +126,28 @@ export default defineComponent({
...
     }
 
+    const undo = async () => {
+      getRegisteringParty()
+    }
+
+    const getRegisteringParty = async () => {
+      var parties: AddPartiesIF = getAddSecuredPartiesAndDebtors.value
+      const regParty = await getRegisteringPartyFromAuth()
+      console.log(regParty)
```
Comment (kialj876) [inline ppr-ui/src/components/parties/RegisteringParty.vue]: remove log pls

**Review by kialj876 — APPROVED** [review]
Looks good 👍

File: ppr-ui/src/components/parties/RegisteringParty.vue (hunk 1 above)
Comment (jordiwes) [reply ppr-ui/src/components/parties/RegisteringParty.vue]: Done

**Review by doug-lovett — APPROVED** [review]
Very nice.

**Pull request merged**